frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -335,6 +336,12 @@ export const SETTINGS_MAP: SettingSection[] = [
                 component: <PreAggregatedTablesSetting />,
                 flag: 'SETTINGS_WEB_ANALYTICS_PRE_AGGREGATED_TABLES',
             },
+            {
+                id: 'web-analytics-opt-int-pre-aggregated-tables-and-api',
+                title: 'New query engine',
```

Comment:
I am intentionally using the name "New query engine" even though the flag is "WEB_ANALYTICS_API" because this is the only way to opt-in right now, and it won't break things for people using the API. 

The next PRs will make this simpler.